public/index.html

```diff
@@ -1,66 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Yelpington</title>
-    <!--Google Font-->
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
-      rel="stylesheet"
-    />
-    <!--leaflet script -->
-    <link
-      rel="stylesheet"
-      href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
-      integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
-      crossorigin=""
-    />
-    <!--leaflet script -->
-    <script
-      src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
-      integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
-      crossorigin=""
-    ></script>
-  </head>
-  <body>
-    <!--CSS-->
-    <link rel="stylesheet" href="yelpington-index.css">
-    <!--Page start. Page container div -->
-    <div id="page">
-      <!--header div containing nav bar and title -->
-      <div id="header">
-        <!--title-->
-        <!-- <h1 id="title">Best of the Mad River Valley (Yelpington)</h1> -->
-        <nav id="nav">
-          <div class="logo">
-           <a href = "/"><h4>The Best of The Mad River Valley</h4></a> 
-            <p>Yelpington</p>
-          </div>
-          <ul id="nav-links">
-            <li></li>
-            <li></li>
-            <li></li>
-            <li> </li>
-            <li></li>
-            <li></li>
-          </ul>
-          <!--burger Icon Tab-->
-          <div class="burger">
-            <div class="line1"></div>
-            <div class="line2"></div>
-            <div class="line3"></div>
-          </div>
```

Comment:
One thing you can do with notes is to leave a comment here saying that it is still a work in progress and that "x, y, z" needs to be completed